README.md

```diff
@@ -14,6 +14,7 @@ The following parameters are always required:
 
  - API_HOST - Domain name of the ApiConnect instance where discovered APIs will be sent.<br /> &nbsp; Example : `us-east.apiconnect.automation.ibm.com`
  - PLATFORM_API_PREFIX - Platform API prefix has a default value of `platform-api` which is the same as the prefix value for ApiConnect on Cloud. It can be changed to match your systems set up if different from the default
+- INSECURE_SKIP_TLS_VERIFY - Set to true when the APIConnect system have self signed cert. In other cases, it can be set to false or ignored. 
```

Comment:
`INSECURE_SKIP_TLS_VERIFY` - If set to `true` the action skips the validity check for the server's certificate. This may be required in the case where your APIConnect system is using self signed certificates. Note: Setting this will make your HTTPS connections insecure.